internal/statemachine/classic_test.go

```diff
@@ -1549,6 +1569,21 @@ func TestFailedPopulateClassicRootfsContents(t *testing.T) {
 		err = stateMachine.populateClassicRootfsContents()
 		asserter.AssertErrContains(err, "Error writing to fstab")
 		ioutilWriteFile = ioutil.WriteFile
+
+		// create an /etc/resolv.conf.tmp in the chroot
+		err = os.MkdirAll(filepath.Join(stateMachine.tempDirs.chroot, "etc"), 0755)
+		asserter.AssertErrNil(err, true)
+		_, err = os.Create(filepath.Join(stateMachine.tempDirs.chroot, "etc", "resolv.conf.tmp"))
```

Comment:
Ok, wanted to propose testing cases where both manual customization and install packages are running to see if the backed up resolv.conf stays the same, but I see this basically catches that test case. Thanks!